test/check-journal

```diff
@@ -178,13 +178,9 @@ ExecStart=/bin/sh -c 'for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 1
 
         m.execute("systemctl start slow10")
 
-        if "222-10" in m.execute("rpmquery systemd || true"):
-            # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=1292805
-            # --follow is broken again, so we need to switch pages explicitly
-            # and hope that the service has started sending out messages in the meantime...
-            b.go("#/?prio=*&start=oldest&service=nonexisting.service")
-            wait_journal_empty()
-            b.go("#/?prio=*&start=oldest&service=slow10.service")
+        b.go("#/?prio=*&start=oldest&service=nonexisting.service")
+        wait_journal_empty()
+        b.go("#/?prio=*&start=oldest&service=slow10.service")
```

Comment:
These three lines are actually the workaround. Their effect is that the page is manually reloaded by visiting a different page first. For a known issue, we can just delete them.